### ISSUE TITLE
Support for Granite text styles

### ISSUE THREAD

**Issue by cassidyjames** [system76/pop-gtk-theme#42]
It'd be handy to add support for the handful of text classes used by elementary OS and Granite-using apps.

It's only a few lines of CSS in the `granite-widgets.css` file in the elementary Stylesheet: https://github.com/elementary/stylesheet/blob/master/gtk-3.0/granite-widgets.css#L373

**Comment by isantop** [comment]
Agreed. I'm not sure if it can be pulled in within the next week, but we can definitely look at it. Alternatively, PRs are always welcome. :)

**Comment by cassidyjames** [comment]
I've started playing with this locally and will PR it if anything useful comes of it. :) It looks like the main differences are some constants that don't exist in Pop and hardcoded font sizes.

**Comment by JosephMcc** [comment]
Using hard coded font sizes can lead to issues. Many users like to tweak sizes on their individual systems which will then break what you are trying to achieve.

**Comment by cassidyjames** [comment]
@JosephMcc of course, but we should be able to set relative font sizes.

**Comment by JosephMcc** [comment]
Do gtk themes properly parse `em`. I don't think I've ever actually tried even though I know gnome-shell and Cinnamon themes do.

**Comment by ryanleesipes** [comment]
I think this should become a higher priority as it looks as though we may adopt some elementary apps as core apps... #idontknowanything
